File: sdkwork-ai-sdk-kotlin/src/main/kotlin/com/sdkwork/ai/Categories.kt
package com.sdkwork.ai

data class Categories(
    val hate: Boolean? = null,
    val sexual: Boolean? = null,
    val violence: Boolean? = null,
    val hateThreatening: Boolean? = null,
    val selfHarm: Boolean? = null,
    val sexualMinors: Boolean? = null,
    val violenceGraphic: Boolean? = null,
    val illegalAct: Boolean? = null,
    val illegalActInstruction: Boolean? = null
)

File: sdkwork-ai-sdk-go/types/categories.go
package types

// Categories
type Categories struct {
	Hate bool `json:"hate"`
	Sexual bool `json:"sexual"`
	Violence bool `json:"violence"`
	HateThreatening bool `json:"hate/threatening"`
	SelfHarm bool `json:"self-harm"`
	SexualMinors bool `json:"sexual/minors"`
	ViolenceGraphic bool `json:"violence/graphic"`
	IllegalAct bool `json:"illegal-act"`
	IllegalActInstruction bool `json:"illegal-act/instruction"`
}
